packages/mtime.1.0.0/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "build": [
+    [
+      "bash",
+      "-c",
+      "#{os == 'windows' ? 'patch -p1 < mtime-1.0.0.patch' : 'true'}"
+    ],
+    [
+      "ocaml",
+      "pkg/pkg.ml",
+      "build",
+      "--pinned",
+      "false",
+      "--with-js_of_ocaml",
+      "false"
```

Comment:
I hardcoded those 2 params. I don't know if we can do better?

Reply:
I think this is fine